__tests__/helpers/expect-os-agnostic.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable jest/valid-expect */
+const normalizeString = (str: string) => str.replace(/\\+/g, '/');
+
+/**
+ * Normalize all data expectations to be OS-agnostic and match against Unix-based filesystems
+ * that have path separators of a forward slash.
+ *
+ */
+export default function expectOSAgnostic(actual: any) {
```

Comment:
@kanadgupta This was a fun hack at first but the more I deal with these last four failures that don't make sense the more I hate this approach I've done.

..maybe we just don't test Windows

![appeardisappear](https://user-images.githubusercontent.com/33762/201505957-7691aa1e-07dc-4972-9bc4-ffcf9f15b440.gif)